roles/lib_openshift/src/class/oc_project.py

```diff
@@ -0,0 +1,183 @@
+# pylint: skip-file
+# flake8: noqa
+
+
+# pylint: disable=too-many-instance-attributes
+class OCProject(OpenShiftCLI):
+    ''' Class to wrap the oc command line tools '''
+    kind = 'namespace'
```

Comment:
@enj had a pull to allow those who can mutate a namespace do it by mutating the project.  I don't recall its status.  As I recall, @liggitt held it on performance concerns.